receiver/loadgenreceiver/config.go

```diff
@@ -25,11 +25,16 @@ type (
 	JsonlFile string
 )
 
-// Config defines configuration for HostMetrics receiver.
+// Config defines configuration for loadgen receiver.
 type Config struct {
 	Metrics MetricsConfig `mapstructure:"metrics"`
 	Logs    LogsConfig    `mapstructure:"logs"`
 	Traces  TracesConfig  `mapstructure:"traces"`
+
+	// Concurrency is the amount of concurrency when sending to next consumer.
+	// The concurrent workers share the amount of workload, instead of multiplying the amount of workload,
```

Comment:
[to reviewer] I've made a design decision here for it to share the workload instead of multiplying the amount of workload. The implication is that when we use this config in otelbench, as concurrency increases, time/op will decrease. If it was implemented otherwise, the time/op will be equal or higher as concurrency increases.